### DISCUSSION TITLE
INVERTEDT is not applied.

### DISCUSSION THREAD

**Discussion by gety9** [manna-harbour/miryoku#69] [Q&A] [answered]
running `qmk flash -c -kb keyboardio/atreus -km manna-harbour_miryoku -e MIRYOKU_ALPHAS=QWERTY -e MIRYOKU_NAV=INVERTEDT -e MIRYOKU_CLIPBOARD=WIN`

qwerty and win settings are applied, but not invertedt, jkl' are still left, down, up, right

using `QMK Firmware 0.15.15`

files in `users\manna-harbour_miryoku` match latest commit https://github.com/manna-harbour/qmk_firmware/commit/7e1430234e802aec1f32c82f018ccb7f68999fc7#diff-5e245d3d2d4dae5c62f05bf62d5b6214af7d1e64badbf516aeb878968058c043

**Comment by manna-harbour** [comment, marked as answer]
That was an earlier commit that added inverted-T with flipped layers only and that is no longer the Miryoku development branch. Please use this branch: https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku for full support.
